Client/Frontend/Browser/URLBarView.swift

```diff
@@ -146,6 +146,10 @@ class URLBarView: UIView {
         return locationView.active
     }
 
+    var canCancel: Bool {
+        return !cancelButton.hidden && cancelButton.enabled
```

Comment:
Not sure the `cancelButton.enabled` is necessary here -- I don't see us modifying its `enabled` state anywhere.